src/constants.ts

```diff
@@ -53,33 +53,33 @@ export const CLASSIC_MODE_MYSTERY_ENCOUNTER_WAVES: [min: number, max: number] =
 export const CHALLENGE_MODE_MYSTERY_ENCOUNTER_WAVES: [min: number, max: number] = [10, 180];
 
 /**
- * Spawn chance: (BASE_MYSTERY_ENCOUNTER_SPAWN_WEIGHT + WIGHT_INCREMENT_ON_SPAWN_MISS * <number of missed spawns>) / MYSTERY_ENCOUNTER_SPAWN_MAX_WEIGHT
+ * Spawn chance: ({@linkcode ME_BASE_SPAWN_WEIGHT} + {@linkcode ME_WEIGHT_INCREMENT_ON_SPAWN_MISS} * <number of missed spawns>) / {@linkcode ME_SPAWN_MAX_WEIGHT}
  */
-export const MYSTERY_ENCOUNTER_BASE_SPAWN_WEIGHT = 3;
+export const ME_BASE_SPAWN_WEIGHT = 3;
 
 /**
  * The divisor for determining ME spawns, defines the "maximum" weight required for a spawn
- * If spawn_weight === MYSTERY_ENCOUNTER_SPAWN_MAX_WEIGHT, 100% chance to spawn a ME
+ * If spawn_weight === {@linkcode ME_SPAWN_MAX_WEIGHT}, 100% chance to spawn a ME
  */
-export const MYSTERY_ENCOUNTER_SPAWN_MAX_WEIGHT = 256;
+export const ME_SPAWN_MAX_WEIGHT = 256;
 
 /**
- * When an ME spawn roll fails, WEIGHT_INCREMENT_ON_SPAWN_MISS is added to future rolls for ME spawn checks.
- * These values are cleared whenever the next ME spawns, and spawn weight returns to BASE_MYSTERY_ENCOUNTER_SPAWN_WEIGHT
+ * When an ME spawn roll fails, {@linkcode ME_WEIGHT_INCREMENT_ON_SPAWN_MISS} is added to future rolls for ME spawn checks.
+ * These values are cleared whenever the next ME spawns, and spawn weight returns to {@linkcode ME_BASE_SPAWN_WEIGHT}
  */
-export const MYSTERY_ENCOUNTER_WEIGHT_INCREMENT_ON_SPAWN_MISS = 3;
+export const ME_WEIGHT_INCREMENT_ON_SPAWN_MISS = 3;
 
 /**
  * Specifies the target average for total ME spawns in a single Classic run.
  * Used by anti-variance mechanic to check whether a run is above or below the target on a given wave.
  */
-export const MYSTERY_ENCOUNTER_AVERAGE_ENCOUNTERS_PER_RUN_TARGET = 12;
+export const ME_AVERAGE_ENCOUNTERS_PER_RUN_TARGET = 12;
 
 /**
- * Will increase/decrease the chance of spawning a ME based on the current run's total MEs encountered vs AVERAGE_ENCOUNTERS_PER_RUN_TARGET
- * Example:
- * AVERAGE_ENCOUNTERS_PER_RUN_TARGET = 17 (expects avg 1 ME every 10 floors)
- * ANTI_VARIANCE_WEIGHT_MODIFIER = 15
+ * Will increase/decrease the chance of spawning a ME based on the current run's total MEs encountered vs {@linkcode ME_AVERAGE_ENCOUNTERS_PER_RUN_TARGET}
+ * @example:
+ * Average-Encounters-Per-Run = 12 (expects avg 1 ME every 10 floors)
```

Comment:
```suggestion
 * Average-Encounters-Per-Run = 17 (expects avg 1 ME every 10 floors)
```

sorry i wasn't super clear last time, but it's better to leave it at 17 I think so the rest of the numbers of the example match with that number, or we have to update all the rest of the comment (ie 12 is not 1 ME every 10 floors, and all the rest of the numbers in the rest of the comment would be different too)